setup/bindir/cloud-setup-databases.in

```diff
@@ -26,6 +26,7 @@ import string
 from optparse import OptionParser
 import subprocess
 import shutil
+import shlex
```

Comment:
@Spaceman1984 what installs `shlex`? It does not look like part of standard py3 library.

Reply:
This is standard python